core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/init/CarbonServerManager.java

```diff
@@ -963,5 +981,51 @@ public void stop() throws Exception {
         artifactsCleanupExec.shutdownNow();
     }
 
-}
+    /**
+     * Set File Handler to print log messages in non-OSGi mode after
+     * EclipseStarter.shutdown() is called.
+     *
+     * @throws IOException If an error occurs while setting handler
+     */
+    private void setJULFileHandler() throws IOException {
+        Handler fileHandler = new FileHandler(LOG_FILES_DIR_PATH);
```

Comment:
new line after the method signature

Reply:
Fixed - https://github.com/wso2/carbon-kernel/pull/2477